### DISCUSSION TITLE
Additional customization settings

### DISCUSSION THREAD

**Discussion by ghost** [bkaankose/Wino-Mail#87] [Ideas]
Good afternoon. I'm very impressed with how you did what Microsoft couldn't do for years the first time. I'm very grateful, thanks to you I won't have to switch to the disgusting Outlook client.
The question itself - could you please include 2 additional customizations?
1 - Adjust the intensity of the effect. Your acrylic is gorgeous, but I would like to be able to make it more blurred.
2 - I'd be incredibly happy if you added the ability to remove the rounding of the interface so it would hormonize better with Windows 10

**Comment by MichalVasut** [comment]
> 1 - Adjust the intensity of the effect. Your acrylic is gorgeous, but I would like to be able to make it more blurred.

I'm not sure what exactly are you talking about, but if it's the blurring when you click on message in the list and it all blurs, then yes. It would be nice to parametrize it - in my case to turn the blurting effect off - it makes the text unreadable and it's really unnecessary.

**Reply by ghost** [reply]
I don't think it's unnecessary, but the intensity of the effect needs to be turned down for better readability and having a slider would be ideal

**Comment by BlueWolfKenji** [comment]
Absolutely backing this up, a tint slider (so the window is stil acrylic, but is darker than the content in the back) would be also awesome. 

THANKS for doing this for us, I absolutely love your app, I wish I had found it sooner.
